### ISSUE TITLE
Add CI

### ISSUE THREAD

**Issue by Leleat** [Leleat/create-gnome-extension#4]
- [x] Check PRs 
    - [x] Check that commits meet angular/conventional-commits rules
- [x] Create a release after pushing a tag
- [x] Push new releases to npm
- [ ] ~~(Maybe?) automatically fill CHANGELOG.md after a release~~ I'm going to use conventional-changelog-cli to generate the CHANGELOG instead of CI/releases
- [ ] ~~Tests~~ Split into separate issue
- [ ] ~~Append link to PR to the commits similiar to marge-bot~~ I am gonna give up on this... I tried to create a workflow that changed the merged commits via gilter-filter-repo. But Github actions don't seem to offer a way to edit the commits before the merge happens, only after. So I would need to allow force-pushing to `main` which I am just gonna skip. This idea was a nice convencience but not that important. I guess I can just manually copy the URL into the merge commit (if I don't forget about it)...
